2023/05.md

```diff
@@ -112,6 +112,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     |   | 2 | 30m | Decorator Metadata for Stage 3 ([slides](https://slides.com/pzuraq/decorator-metadata-for-stage-3), [spec text](https://github.com/pzuraq/ecma262/pull/10)) | Chris Hewell Garrett |
     |   | 2 | 45m | [Float16Array](https://github.com/tc39/proposal-float16array) for stage 3 ([slides](https://docs.google.com/presentation/d/1gBVaE4KV9JhRxG_V4_xjLW7P18lnKCDJEIktpi6eIxs/edit), [spec](http://tc39.es/proposal-float16array/)) | Kevin Gibbons |
     |   | 2 | 45m | 🔒 [Source Phase Imports](https://github.com/tc39/proposal-import-reflection) for stage 3 ([slides](https://docs.google.com/presentation/d/1lpJBCe6cgBa5MBtD_Bhfyh92EXcJS72LbStnaoQ4cxo/edit), [spec](https://github.com/tc39/proposal-import-reflection/pull/36)) | Luca Casonato & Guy Bedford |
+    |   | 2 | 45m | ⌛️ [Async iterator](https://github.com/tc39/proposal-async-iterator-helpers) design/tradeoffs for stage 3 ([slides](https://docs.google.com/presentation/d/10y7dAh6LaBqVEV5E7Wb8kDZ9XBkfqTY-pC_JUCBVk30)) | Kevin Gibbons |
```

Comment:
"for stage 3" can be confusing here

```suggestion
    |   | 2 | 45m | ⌛️ [Async iterator](https://github.com/tc39/proposal-async-iterator-helpers) stage 2 feedback needed on design/tradeoffs ([slides](https://docs.google.com/presentation/d/10y7dAh6LaBqVEV5E7Wb8kDZ9XBkfqTY-pC_JUCBVk30)) | Kevin Gibbons |
```

Reply:
oh whoops, definitely did not mean to include that